package.json

```diff
@@ -13,6 +13,7 @@
     "lint-tools": "eslint --ext js 'tools/**/*.js'",
     "lint-common": "eslint --ext js 'common/**/*.js'",
     "lint": "npm run lint-client && npm run lint-server && npm run lint-tools && npm run lint-common",
+    "stylelint": "npx stylelint '**/*.css'",
```

Comment:
For consistency: rename this entry `lint-style`, put it right below `lint-common` and chain a call to it in `lint` with `&& npm run lint-style`